test/src/edu/stanford/nlp/objectbank/LineIteratorTest.java

```diff
@@ -4,32 +4,44 @@
 import java.util.Iterator;
 
 import org.junit.Assert;
+import org.junit.Before;
```

Comment:
I wasn't sure what is being improved here - git commit comments would be a good place to explain that!  I merged the rest of the changes in this commit, though.